root/static/scripts/common/i18n/expand2.js

```diff
@@ -214,9 +214,6 @@ export const createVarSubstParser = <T, V>(
   },
 );
 
-export const parseStringVarSubst: Parser<string | NO_MATCH, mixed> =
```

Comment:
I see this is moving to `expand2react` / `expand2text,` what's the reasoning? It's not possible to support both types? It's more clear? Both?

Reply:
At one point it was to resolve some type errors which I also thought were cause by the 0.217.0 upgrade, but it seems I can revert this without breaking anything, so I did that.